tests/cancun/eip4788_beacon_root/test_beacon_root_contract.py

```diff
@@ -278,14 +278,10 @@ def test_invalid_beacon_root_calldata_value(
     )
 
 
-@pytest.mark.parametrize("timestamp", [12])
 @pytest.mark.valid_from("Cancun")
 def test_beacon_root_selfdestruct(
     blockchain_test: BlockchainTestFiller,
-    beacon_root: bytes,
-    timestamp: int,
     pre: Alloc,
-    tx: Transaction,
     post: Dict,
 ):
```

Comment:
We indeed don't need any of `timestamp`, `tx` or `beacon_root`. We also don't need `post`, as the test body overwrites the value from the function argument. Removed in 129f71c35d8.

However, removing `tx` modifies the generated fixture. This is because the `tx` fixture has the side effect of adding an EOA to the pre-state (to fund the sender). This is unrequired by this test and is worth cleaning out.

So while Sam's original change was safe, it took a moment to verify it.